tests/tasks/test_fitbit_poll.py

```diff
@@ -158,22 +158,22 @@ async def test_fitbit_poll_activity(  # noqa PLR0913
 
     # Mock fitbit endpoint to return no sleep data
     respx_mock.get(
-        url=f"{settings.fitbit_base_url}1.2/user/-/sleep/date/2023-01-23.json",
+        url=f"{settings.fitbit_oauth_settings.base_url}1.2/user/-/sleep/date/2023-01-23.json",
     ).mock(Response(status_code=200, json={"sleep": []}))
 
     # Mock fitbit endpoint to return some activity data
     respx_mock.get(
-        url=f"{settings.fitbit_base_url}1/user/-/activities/list.json",
+        url=f"{settings.fitbit_oauth_settings.base_url}1/user/-/activities/list.json",
     ).mock(Response(status_code=200, json=scenario.input_mock_fitbit_response))
 
     # Mock an empty ok response from the slack webhook
-    slack_request = respx_mock.post(f"{settings.slack_webhook_url}").mock(
-        return_value=Response(200)
-    )
+    slack_request = respx_mock.post(
+        f"{settings.secret_settings.slack_webhook_url}"
+    ).mock(return_value=Response(200))
 
     if scenario.settings_override:
         for key, value in scenario.settings_override.items():
-            monkeypatch.setattr(settings, key, value)
+            monkeypatch.setattr(settings.app_settings.fitbit.activities, key, value)
```

Comment:
see how to override settings at a higher level.

Or: rename settings_override to something more specific.